src/elo/utils/matchBracket.js

```diff
@@ -0,0 +1,29 @@
+function matchBracket(rating, totalRankedGamesPlayed) {
+    const provisionalGames = 20;
+    const bronzeBase = 1300;
+    const silverBase = 1400;
+    const goldBase = 1550;
+    const platBase = 1700;
+    const diamondBase = 1800;
+    const championBase = 1900;
```

Comment:
These should be capital constants. Please also lift these up out of the function!